### DISCUSSION TITLE
“BitClout powered” node logo

### DISCUSSION THREAD

**Discussion by marnimelrose** [deso-protocol/dips#94] [Q&A]
I think it would be cool and helpful 
If￼ there was a graphic that would check if a node is using BitClout identity and if so then the node would be able to display the BitClout powered￼ logo￼

**Comment by tijno** [comment]
Any suggestions on how this could be implemented ?

And how it can be secured against faking / abuse ?

Eg a "kite mark" is often seen as a good idea - but is then abused bu scammer who use similar likeness images or copies to imply trust. 

Lastly - any user trying to login will see its using identity.

I think its important users get used to checking the url of the identity popup window for security reasons.
